lib/common.js

```diff
@@ -395,6 +395,105 @@ function promptYesNo(opts_, cb) {
     }
 }
 
+/**
+ * Interactively prompt the user for a confirmation string.
+ *
+ *      cb(true)       The user entered the confirmation string.
+ *      cb(false)      The user ^C'd.
+ */
+function promptConfirm(opts_, cb) {
```

Comment:
ooc: what's the need for this function as opposed to using node's readline or similar? It seems surprising to me that we're doing our own terminal/character handling here, so it might be worth explaining the reasoning in the comment.

Reply:
I adapted that from the existing promptYesNo that I've had kicking around in repos since days of yore.

Interestingly, Tim relatedly asked about promptYesNo doing its own char handling at https://github.com/joyent/sdc-manta/pull/60#pullrequestreview-383946859
I think using readline is probably a good idea. However, *right now*, I don't particularly want to go through testing the various potential use cases (admittedly use cases that are for promptYesNo and in other repos) -- see https://gist.github.com/trentm/3bec297455cb1786be3e3264d551ff82 for some gory details.

A comment is a good idea. I'll add that.

Reply:
Done in commit 64b0a3b